hueniversitypy/theme_ubc.py

```diff
@@ -1,21 +1,75 @@
 def theme_ubc(palette='blue'):
```

Comment:
```suggestion
def theme_ubc():
```
Since you only have one colour scheme I don't think you need a `palette` argument anymore.